src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arrResult = [];
+  const STATE_PREVIOS = { ...state };
+
+  for (const OBJ_ACTION of actions) {
```

Comment:
The variable name `OBJ_ACTION` does not follow naming conventions. Consider using `action` or `currentAction` for clarity.